modin/pandas/dataframe.py

```diff
@@ -2180,6 +2180,59 @@ def is_dtype_instance_mapper(column, dtype):
         ]
         return self.drop(columns=self.columns[indicate], inplace=False)
 
+    def sem(
```

Comment:
Implementation of the method seems to me identical for Series and Dataframe.

Can implementation be defined in `backends/pandas/query_compiler`?

Reply:
The implementation is specific for `Series` and `DataFrame`. Because we treat `Series` as `DataFrame` in the backend, the implementation can't be defined in `backends/pandas/query_compiler`.